ex1/ex1.js

```diff
@@ -6,3 +6,25 @@
  *     fn(",,,") === 2
  *     fn("hel,lowo,rld") === -1
  */
+
+/////////////////////////////////////////////////////////////////////////////
+// getThirdCommaPosition accepts a string and returns the index of the third comma in that string /  no comma return -1
+function getThirdCommaPosition(str,searcVal) {
```

Comment:
searchVal was not requested in the question, but fine.
BTW - searcVal --> searchVal (with h).